tests/integration/util/config.rs

```diff
@@ -17,6 +17,11 @@ pub struct Config {
     bucket: Option<String>,
     enabled_features: Vec<TestFeature>,
     data_timeout: String,
+    capella_conn_string: Option<String>,
+    capella_username: Option<String>,
```

Comment:
Do we need to duplicate username and password, can we not just use the existing config properties for that? Could we maybe add a new cluster_type for capella or something?